### PR TITLE
Seed

### PR DESCRIPTION
**Pull request by paprikatie**
Finish seeding functions + image urls from S3 bucket (HEAD -> seed, origin/seed)
Set up webpack, install dependencies, connect mongodb with mongoose, write seeding functions
Write server GET and finish populating database with dummy data from seed.js

### CODE REVIEW

File: server/seed.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+};
+
+const randReviews = () => {
+  let num = random(0, 1000, true);
+  if (num === 0) {
+    return 0;
+  } else if (num === 1) {
+    return 1;
+  }
+  return ' (' + num + ')';
```
Comment (emmanuelmota) [inline server/seed.js]: You could possibly use template literals here ?

File: server/seed.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  ' in the Hills'
+];
+
+//pull random from array of names
+const randTitle = () => {
+  let index = random(0, titles.length, true);
+  let index1 = random(0, adjectives.length, true);
+  let index2 = random(0, nouns.length, true);
+  let index3 = random(0, add_ons.length, true);
+  return titles[index] + ' - ' + adjectives[index1] +
```
Comment (emmanuelmota) [inline server/seed.js]: Should be a dot instead of a dash

**Review by emmanuelmota — CHANGES_REQUESTED** [review]
Looking good so far! Just a few notes.

**Comment by paprikatie** [comment]
Made the changes you requested!

**Review by emmanuelmota — APPROVED** [review]

**Pull request merged**